front/app/modules/commercial/insights/admin/components/StyledTextComponents.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
```

Comment:
If you added those to be reusable, do make sure you communicate it and hopefully move them them to the library, I don't know about now but we used to have plenty of those "Section", "SectionTitle" "AdminTitle" components all over the codebase.

Reply:
These are Insights specific but I do feel we have so much of that. Will prioritise creating some Typography components in the component library so we can standartise this at least a little bit.